src/main/resources/templates/repo/kinesis-log-streams.json

```diff
@@ -14,21 +14,62 @@
 		"bufferFlushInterval": 900,
 		"tableDescriptor": {
 			"name": "fileDownloadsRecords",
-			"columns": {
-				"userId": "bigint",
-				"timestamp": "timestamp",
-				"projectId": "bigint",
-				"fileHandleId": "string",
-				"associateType": "string",
-				"associateId": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"year": "string",
-				"month": "string",
-				"day": "string"
-			}
+			"description" : "The table contain records of all downloads of Synapse.The events are recorded only after the pre-signed url for requested download entity is generated.",
+			"columns": [{
+				"name": "userId",
+				"type" : "bigint",
+				"comment" : "The id of the user who downloaded the file."
+			},
+			{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the download event is pushed to the queue for recording, after generating the Pre-signed url."
+			},
+			{
+				"name": "projectId",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the project where the downloaded entity resides. Applicable only for FileEntity and TableEntity"
+			},
+			{
+				"name": "fileHandleId",
+				"type" : "string",
+				"comment" : "The unique identifier of the file handle."
+			},
+			{
+				"name": "associateType",
+				"type" : "string",
+				"comment" : "The type of the Synapse object that wraps the file."
+			},
+			{
+				"name": "associateId",
+				"type" : "string",
+				"comment" : "The unique identifier of the Synapse object ( with prefix syn) that wraps the file."

```

Comment:
The syn prefix is stripped out in the rest of the tables, should this be consistent?

Reply:
These are existing old tables and have syn id. The tables in this file  is not related to new dataware-house changes.